chemprop/models/model.py

```diff
@@ -68,6 +69,7 @@ def __init__(
         init_lr: float = 1e-4,
         max_lr: float = 1e-3,
         final_lr: float = 1e-4,
+        input_scalers: dict[str, StandardScaler] | None = None,
```

Comment:
what does this do?

Reply:
I see... I wonder if there's a better way for us to couple this information alongside the model artifacts (i.e., the architecture and weights) rather than hacking it into a model checkpoint. I don't have any ideas right now, but something to keep in mind.

Reply:
I also wish there was a better way to save and distribute the input scalers than to treat them like hyperparameters of the model. 
An idea would be to put the input scalers in its own file instead of the model files, but I have a strong preference to not do this so that when you share a model with someone, there is only one file. 
Another idea is to use `on_save_checkpoint(self, checkpoint)` to add the scalers to the checkpoint file when it is created. But this still requires the input scalers to be given to the model and saved as an attribute to self, which to me makes it equivalent to what we already do. 

The essential issue boils down to:
1. We want to save the scalers in the checkpoint file.
2. The lightning trainer handles making the checkpoint file.
3. The lightning trainer only takes a model and a dataset as arguments.
4. Therefore we need to input the input scalers in the model.